hbase-coprocessor/src/main/java/org/apache/hadoop/hbase/coprocessor/ObserverContextImpl.java

```diff
@@ -87,19 +89,5 @@ public boolean shouldBypass() {
     return Optional.ofNullable(caller);
   }
 
-  /**
-   * Instantiates a new ObserverContext instance if the passed reference is <code>null</code> and
-   * sets the environment in the new or existing instance. This allows deferring the instantiation
-   * of a ObserverContext until it is actually needed.
-   * @param <E> The environment type for the context
-   * @param env The coprocessor environment to set
-   * @return An instance of <code>ObserverContext</code> with the environment set
-   */
-  @Deprecated
```

Comment:
We are good to remove this in 2.x releases?

Reply:
Sorry for confusion @virajjasani ... This exploratory work is against branch-2 but target is master branch. We cannot remove these methods as you suggest in branch-2 but we could in master. Odd is that these methods are deprecated but without an alternative.... I need to provide something.